src/TypeSpec.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -1,203 +1,121 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-import { createSdkContext } from "@azure-tools/typespec-client-generator-core";
-import {
-    EmitContext,
-    Program,
-    logDiagnostics,
-    resolvePath
-} from "@typespec/compiler";
+import { EmitContext, Program, resolvePath } from "@typespec/compiler";
```

Comment:
How we are going to handle the Azure exclusive logic which would not go into http-client-csharp?

Reply:
As discussed stage 1: all decorators(include azure-specific decorator) will be handled in `http-client-csharp`, so we does not need specific Azure logic in emitter. 
in future, (stage 2 or later stage) tcgc will separate azure/non-azure libraries, and more complex emitter framework (under-discussing) designed, we then will handle this.